packages/foreman/foreman/foreman.spec

```diff
@@ -65,6 +65,7 @@ Requires: %{?scl_prefix}rubygem(will_paginate) >= 3.1.7
 Requires: %{?scl_prefix}rubygem(will_paginate) < 4
 Requires: %{?scl_prefix}rubygem(ancestry) >= 2.0
 Requires: %{?scl_prefix}rubygem(ancestry) < 4
+BuildRequires: %{?scl_prefix}rubygem(bigdecimal)
```

Comment:
Can you move this down to the BuildRequires section. This should also be `%{?scl_prefix_ruby}` for EL7. I would also make this a Requires as well as we are highly likely to need this at runtime.